pkg/dispatchcli/cmd/update_baseimage.go

```diff
@@ -0,0 +1,98 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2018 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/vmware/dispatch/pkg/dispatchcli/i18n"
+	"github.com/vmware/dispatch/pkg/image-manager/gen/client/base_image"
+	"github.com/vmware/dispatch/pkg/image-manager/gen/models"
+)
+
+var (
+	updateBaseImageLong    = "Updates a base image from a given json representation"
+	updateBaseImageExample = `{
+		"dockerUrl": "vmware/dispatch-openfaas-nodejs6-base:0.0.3-dev1",
+		"groups": null,
+		"language": "nodejs6",
+		"name": "nodejs6-base",
+		"public": true,
+		"reason": null,
+		"tags": [
+			{
+				"key": "role",
+				"value": "test"
+			}
+		]
+	}`
+	imageURL string
+)
+
+func CallUpdateBaseImage(input interface{}) error {
+	baseImage := input.(*models.BaseImage)
+	params := base_image.NewUpdateBaseImageByNameParams()
+	params.BaseImageName = *baseImage.Name
+	params.Body = baseImage
+	_, err := imageManagerClient().BaseImage.UpdateBaseImageByName(params, GetAuthInfoWriter())
+
+	if err != nil {
+		return formatAPIError(err, params)
+	}
+
+	return nil
+}
+
+func NewCmdUpdateBaseImage(out io.Writer, errOut io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "base-image BASE_IMAGE_NAME [--image-url IMAGE_URL] [--language LANGUAGE]",
+		Short:   i18n.T("Update base image"),
+		Long:    updateBaseImageLong,
+		Example: updateBaseImageExample,
+		Args:    cobra.MinimumNArgs(1),
```

Comment:
You always use `args[0]` in `updateBaseImage()`, So `ExactArgs` seems more suitable here.